assets/webidl.css

```diff
@@ -14,7 +14,7 @@ pre.idl {
 .idlHeader {
   display: block;
   width: 150px;
-  background: #90b8de;
+  background: #8ccbf2;
```

Comment:
Same as border color or IDL block

Reply:
Before:
![image](https://user-images.githubusercontent.com/8426945/76206629-6324fd00-6222-11ea-8643-e7017a3b97bf.png)

After:
![image](https://user-images.githubusercontent.com/8426945/76206644-6ddf9200-6222-11ea-9e75-acfa3f374e04.png)

Reply:
hehe, 👓... totally see the difference!

Reply:
in all seriousness, I did prefer the white... but can live with the dark.